app/models/service.rb

```diff
@@ -108,13 +108,13 @@ def cinstance
     end
   end)
 
-  validates :tech_support_email, :admin_support_email, :credit_card_support_email, format: { with: /.+@.+\..+/, allow_blank: true }
```

Comment:
Considering that we are not removing the attributes yet at all, imo we should keep the validation until we decide to completely remove these attributes from the code. Or if we decide to remove the attributes completely from the code and not only what the customer can interact with, then you do well in removing the validations but then we should also change this line:
https://github.com/3scale/porta/blob/245555bbef2f265cec806a8a6f40152a1ce57e66/app/models/service.rb#L46
for:
```ruby
super.reject { |column| %w[buyer_can_see_log_requests tech_support_email admin_support_email].includes?(column.name) }
```
[The reason](https://medium.com/klaxit-techblog/zero-downtime-migrations-with-activerecord-47528abe5136) is this tells ActiveRecord to ignore these column from its cache, this way later on when we remove the attributes from the DB, the migration in SaaS may be done by Operations manually and if the code is still using it (in this case Rails having it in its cache) while they are removing it from DB, it will crash, [as it happened to me](https://github.com/3scale/porta/pull/545) for the `deleted_at` of `Account`.

Reply:
I'll take the second option